content/cpp/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'round()'
+Description: 'Returns the integral value that is nearest to the argument, with halfway cases rounded away from zero.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `round()` function returns the integral value that is nearest to the argument, with halfway cases rounded away from zero.
+
+## Syntax
+
+The `cmath` library must be added at the top of the file.
+
+```cpp
+std::round(double num);
+```
+
+Argument must be a `double`/`float`/`long double`, and the return value will be the type of `integer`.
+
+## Example
+
+```cpp
+double num, result;
+
+result = std::round(num);
+```
+
```

Comment:
This example could be expanded a bit more. Can we assign a numeric value to `num` before passing into `round()`?